resources_portal/models/material_request.py

```diff
@@ -0,0 +1,38 @@
+from django.db import models
+
+from .material import Material
+from .user import User
+
+
+class MaterialRequest(models.Model):
+    class Meta:
+        db_table = "material_requests"
+        get_latest_by = "created_at"
+
+    objects = models.Manager()
+
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    material = models.ForeignKey(
+        Material, blank=False, null=False, on_delete=models.CASCADE, related_name="requests"
+    )
+
+    requester = models.ForeignKey(
+        User, blank=False, null=False, on_delete=models.CASCADE, related_name="material_requests"
+    )
+
+    assigned_to = models.ForeignKey(
+        User, blank=False, null=False, on_delete=models.CASCADE, related_name="assigned_to_set"
```

Comment:
What about `User.inbound_requests` and `User.outbound_requests` for the related names? Is that as clear as it sounds to me?

Reply:
hmm what would be the difference between `inbound_requests` and `outbound_requests`?

I was thinking of using `assignments` for this one as @davidsmejia suggested

Reply:
Mmm yeah I like that!